### PR TITLE
telco5g: Adding cnftest BM job - 4.14/4.15 on cnf-features-deploy branches

### PR DESCRIPTION
**Pull request by Sandeepyadav93**
With this Pr, Adding  cnftest BM job - for 4.14/4.15
branches of cnf-features-deploy repo.

[CNF-10753](https://issues.redhat.com//browse/CNF-10753)

### CODE REVIEW

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.14.yaml [hunk 1]
```diff
@@ -110,6 +110,31 @@ tests:
...
+  as: e2e-telco5g-sno-cnftests
+  cluster: build05
+  optional: true
+  steps:
+    env:
+      E2E_TESTS_CONFIG: |
+        CNF_E2E_TESTS=true
+        CNF_ORIGIN_TESTS=false
+      T5CI_JOB_TYPE: sno-cnftests
+      T5CI_VERSION: "4.16"
```
Comment (sshnaidm) [inline ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.14.yaml:136]: 4.14

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-master-e2e-telco5g-sno-cnftests pull-ci-openshift-kni-cnf-features-deploy-release-4.15-e2e-telco5g-sno-cnftests

File: ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.14.yaml (hunk 1 above)
Comment (Sandeepyadav93) [reply ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-release-4.14.yaml]: thanks

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-release-4.14-e2e-telco5g-sno-cnftests pull-ci-openshift-kni-cnf-features-deploy-release-4.15-e2e-telco5g-cnftests

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-release-4.14-e2e-telco5g-cnftests

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-release-4.15-e2e-telco5g-cnftests

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-release-4.14-e2e-telco5g-cnftests

**Comment by Sandeepyadav93** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-release-4.14-e2e-telco5g-cnftests pull-ci-openshift-kni-cnf-features-deploy-release-4.15-e2e-telco5g-cnftests

**Review by SchSeba — APPROVED** [review]
/lgtm
/approve

**Pull request merged**